administration/users.rst

```diff
@@ -1,57 +1,130 @@
 Users
 #####
 
-In this section you can create, edit and access information of |omv| users and groups.
+User management in |omv| is supported by the users mnagement of the operating system 
+it runs on.
+
+However, |omv| also maintains control over these users, **so management is a team effort
+between the Debian operating system and the OMV software system** internal database.
+
+**Users** management is divided into three subsections: ``Settings``, ``Users`` and ``Groups``.
+
+Groups means access for multiple users to multiple shared resources.
+
+An user can log into the |webui| to see their own profile information.
+
+With this section we can manage also permissions of shared resources.
+Unlike user permissions, group permissions allow you to define access
+for multiple users to the same resource.
+
+About home user
+^^^^
+
+Due to the nature explained, users are supposed to have their own private place for
+files that is called "*home*", depending on the type of service, automatically becomes 
+a personal private shared location.
+
+This place is configured in the ``Settings`` subsection of the |webui| interface 
+of |omv|, as *User home directory*
+
+Settings
+========
+
+This is the first subsection of the **Users** management section of |webui|.
+
+Allows to select a |sf| as root for *home* place for new users created in the
+|webui|.
+
+* If you dont enable *User home directory*, when created an user:
+
+  * Will not have a private place for files if you use the |omv| |webui|
+  * Will have a private place for files if you use :command:`useradd` command
+  * The "skel" templates from Debian only will be used in last case
+
+* If you already enable *User home directory*, when created an user:
+
+  * Will have a private place for files no matter way you use to create!
+  * The "skel" templates from Debian will be used always
+
+* If you disabled and re-enabled *User home directory* and viceversa:
+
+  * Previously existing users created before enabling/disabling this setting 
+    will not have their *home* directory moved to the new location.
+  * Previously existing users  created before enabling/disabling this setting
+    can change *home* by editing :file:`/etc/passwd` to point them to the new 
+    location.
+  * Also existing users data in default linux location :file:`/home`
+    or previously existing *home* has to be moved manually too.
 
 User
 ====
 
-Here you can create or modify user information and configure the user home folders.
+This allows to *Create* or *Edit* users as well as their *Permissions*, there is a
+special option that allows batch loading called *Import*.
 
-Add
-^^^^
+The options of *Create* and *Import* are under the "+" button of the configuration panel.
```

Comment:
ok, i was just improved because this documentation do not have any images.. mayor documentations provides images to maintain course of the guidance.. just see the docs of mayor private products.. 

documentation on companies are product of studding using sociological and human resource management